.github/workflows/ci-e2e.yaml

```diff
@@ -14,6 +14,8 @@ on:
         description: 'Base url for backend API'
         required: true
         default: 'https://api.watch.kausal.tech/v1'
+      schedule:
+        - cron: '0 22 * * *' # 22:00 UTC, 01:00 EEST
```

Comment:
I'm definitely no expert on GH action scripts, but I _think_ `schedule` needs to be at the same level as `workflow_dispatch` since they're both separate events, i.e.

```yaml
name: E2e-tests
on:
  schedule:
    - cron: '0 22 * * *' # 22:00 UTC, 01:00 EEST
  workflow_dispatch:
```